cli/src/stake.rs

```diff
@@ -487,6 +500,13 @@ impl StakeSubCommands for App<'_, '_> {
                         .help("Seed for address generation; if specified, the resulting account \
                                will be at a derived address of SPLIT_STAKE_ACCOUNT")
                 )
+                .arg(
+                    Arg::with_name("force")
+                        .long("force")
```

Comment:
This should use the new name too, right?
```suggestion
                    Arg::with_name("skip_balance_checks")
                        .long("skip-balance-checks")
```